dlt/common/json/__init__.py

```diff
@@ -12,6 +12,11 @@
 except ImportError:
     PydanticBaseModel = None  # type: ignore[misc]
 
+try:
+    from pydantic import AnyUrl as PydanticAnyUrl
+except ImportError:
+    PydanticAnyUrl = None  # type: ignore[misc]
```

Comment:
I cargo-culted this from above - I am assuming this is for the case when the Pydantic classes are not available and meant to gracefully degrade?